src/pybind/mgr/dashboard/frontend/src/app/shared/components/cd-label/cd-label.component.html

```diff
@@ -0,0 +1,5 @@
+<span #contentWrapper
+      class="badge badge-background-primary"
+      [ngClass]="'cd-label-color-' + [color_class]">
```

Comment:
When the input arg is mostly text you can also use this syntax:
```suggestion
      ngClass="cd-label-color-{{ color_class }}">
```